infrasim/log.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import os
+from infrasim.config import infrasim_log_file, infrasim_logdir
+
+EXCEPT_LEVEL_NUM = 35
+
+
+def EXCEPTION(self, message, *args, **kws):
+    # Yes, logger takes its '*args' as 'args'.
+    self._log(EXCEPT_LEVEL_NUM, message, args, **kws)
+
+
+class logger_type:
+    def __init__(self):
+        self.model = 'Model'
+        self.qemu = 'Qemu'
+        self.ipmi_sim = 'Ipmi_sim'
+        self.socat = 'Socat'
+        self.ipmi_console = 'Ipmi-console'
+        self.racadm = 'Racadmsim'
+        self.cmd = 'Cmd'
+        self.config = 'Config'
+        self.environment = 'Environment'

```

Comment:
@WinnieXU1 , `enumeration` is the language of programmer :speak_no_evil: , see if below example make sense:

    from enum import Enum

    class test(Enum):
        a = 1
        b = 2

    print test.a
    print repr(test.a)
    print test.a.name
    print test.a.value